# HG changeset patch
# User Rik <rik@octave.org>
# Date 1392934094 28800
#      Thu Feb 20 14:08:14 2014 -0800
# Branch gui-release
# Node ID 554be77a60fb9a27bce21ff3e38547126d843d18
# Parent  523878f765183ce1bee93a783ed575036f392b27
build: Add FT2_CFLAGS to CPPFLAGS required for new Qt graphics.

* libgui/graphics/module.mk: Add FT2_CFLAGS to
graphics_libgui_graphics_la_CPPFLAGS.

* libgui/src/module.mk: Add FT2_CFLAGS to src_libgui_src_la_CPPFLAGS.

diff --git a/libgui/graphics/module.mk b/libgui/graphics/module.mk
--- a/libgui/graphics/module.mk
+++ b/libgui/graphics/module.mk
@@ -100,16 +100,17 @@ graphics_libgui_graphics_la_SOURCES = \
   graphics/ToolBar.cc \
   graphics/Utils.cc \
   graphics/gl-select.cc
 
 nodist_graphics_libgui_graphics_la_SOURCES = $(octave_gui_graphics_MOC) $(octave_gui_graphics_RC)
 
 graphics_libgui_graphics_la_CPPFLAGS = \
   $(AM_CPPFLAGS) \
+  $(FT2_CFLAGS) \
   $(FONTCONFIG_CPPFLAGS) \
   @OCTGUI_DLL_DEFS@ \
   @QT_CPPFLAGS@ \
   -Igraphics -I$(srcdir)/graphics \
   -I$(top_srcdir)/liboctave/cruft/misc \
   -I$(top_srcdir)/liboctave/array \
   -I$(top_builddir)/liboctave/numeric -I$(top_srcdir)/liboctave/numeric \
   -I$(top_builddir)/liboctave/operators -I$(top_srcdir)/liboctave/operators \
diff --git a/libgui/src/module.mk b/libgui/src/module.mk
--- a/libgui/src/module.mk
+++ b/libgui/src/module.mk
@@ -168,16 +168,17 @@ src_libgui_src_la_SOURCES = \
   src/find-files-model.cc \
   src/workspace-model.cc \
   src/workspace-view.cc
 
 nodist_src_libgui_src_la_SOURCES = $(octave_gui_MOC) $(octave_gui_RC)
 
 src_libgui_src_la_CPPFLAGS = \
   $(AM_CPPFLAGS) \
+  $(FT2_CFLAGS) \
   $(FONTCONFIG_CPPFLAGS) \
   @OCTGUI_DLL_DEFS@ \
   @QT_CPPFLAGS@ \
   -I$(srcdir)/qterminal/libqterminal \
   -Isrc -I$(srcdir)/src \
   -I$(srcdir)/src/m-editor \
   -I$(srcdir)/src/qtinfo \
   -I$(srcdir)/graphics \
